get-median-wasm/Makefile

```diff
@@ -1,16 +1,17 @@
 
-# TODO: Environment setup like rustup target add wasm32-wasi
-
 SRC_WASM = ./
+SRC_RST = ./src/
 BIN = ../bin/
 APP_WASM = get_median.wasm
 BIN_WASM = $(addprefix $(BIN), $(APP_WASM))
+FILES_RUST_F= $(wildcard $(SRC_RST)*.rs) # Wildcard function used, no need to specify the rust files. Safe as we don't compile the rust files with the makefile.
+TOML = Cargo.toml
 all: $(BIN_WASM)
 
-$(BIN_WASM):
+$(BIN_WASM): $(FILES_RUST_F) $(TOML)
 	@echo "\033[32mStarting cargo to build the wasm binary...\033[0m"
-	@cd $(SRC_WASM) && cargo build --target wasm32-wasi --release
-	@cp $(SRC_WASM)/target/wasm32-wasi/release/get_median_wasm.wasm $(BIN)/$(APP_WASM)
+	@cd $(SRC_WASM) && cargo build --target wasm32-unknown-unknown --release
+	@cp $(SRC_WASM)/target/wasm32-unknown-unknown/release/get_median_wasm.wasm $(BIN)/$(APP_WASM)
 
 clean:
 	@cargo clean
```

Comment:
Not for this crate since we are building a standalone binary. We should commit the lock file as well. See: https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html